packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,389 @@
+    setBottomSheetVisible(false)
+  }
+
+  const onPressHelp = () => {
+    setHelpDialogVisible(true)
+  }
+
+  const onPressNavigationButton = () => {
+    if (!dappSelected) {
+      // Should never happen
+      Logger.warn(TAG, 'Internal error. There was no dapp selected')
+      return
+    }
+    dispatch(openUrl(dappSelected.dappUrl, true, true))
+    setBottomSheetVisible(false)
+  }
+
+  return (
+    <SafeAreaView style={styles.safeAreaContainer}>
+      <CustomHeader
+        left={<BackButton />}
+        title={t('dappsScreen.title')}
+        right={
+          <TopBarIconButton
+            icon={<Help />}
+            onPress={onPressHelp}
+            style={styles.helpIconContainer}
+          />
+        }
+      />
+      <BottomSheet isVisible={isBottomSheetVisible} onBackgroundPress={onCloseBottomSheet}>
+        <View>
+          <View style={styles.bottomSheetHeader}>
+            <TopBarIconButton icon={<QuitIcon />} onPress={onCloseBottomSheet} />
+          </View>
+          <View style={styles.centerContainer}>
+            <Text style={{ ...fontStyles.h2, textAlign: 'center', paddingVertical: 16 }}>
+              {t('dappsScreenBottomSheet.title', { dappName: dappSelected?.name })}
+            </Text>
+            <Text style={{ ...fontStyles.regular, textAlign: 'center', flex: 1 }}>
+              {t('dappsScreenBottomSheet.message')}
+            </Text>
+            <Button
+              style={styles.bottomSheetButton}
+              onPress={onPressNavigationButton}
+              text={t('dappsScreenBottomSheet.button', { dappName: dappSelected?.name })}
+            />
+          </View>
+        </View>
+      </BottomSheet>
+
+      <Dialog
+        title={t('dappsScreenHelpDialog.title')}
+        isVisible={isHelpDialogVisible}
+        secondaryActionText={t('dappsScreenHelpDialog.dismiss')}
+        secondaryActionPress={() => setHelpDialogVisible(false)}
+      >
+        {t('dappsScreenHelpDialog.message')}
+      </Dialog>
+
+      <ScrollView style={styles.scrollContainer}>
+        <View style={styles.descriptionContainer}>
+          <Text style={{ ...fontStyles.h1, flex: 1 }}>{t('dappsScreen.message')}</Text>
+          <View style={styles.descriptionImage}>
+            <DappsExplorerLogo />
+          </View>
+        </View>
+        <>
+          {loading && (
+            <View style={styles.centerContainer}>
+              <ActivityIndicator
+                style={styles.loadingIcon}
+                size="large"
+                color={colors.greenBrand}
+                testID="DAppExplorerScreen/loading"
+              />
+            </View>
+          )}
+          {!loading && error && (
+            <View style={{ ...styles.centerContainer, marginTop: 96 }}>
+              <Text style={fontStyles.regular}>{t('dappsScreen.errorMessage')}</Text>
+            </View>
+          )}
+          {!loading && !error && result && (
+            <SectionList
```

Comment:
I'm not sure about having the `SectionList` inside the `ScrollView` since they are both scrollable, I think the containing `ScrollView` should be just a View? I'm not sure how the interaction works though, mentioning it just in case.

Reply:
I'll try removing the ScrollView and see how that works :)